cmd/obslytics/export.go

```diff
@@ -66,20 +67,44 @@ func registerExport(m map[string]setupFunc, app *kingpin.Application) {
 			if err != nil {
 				return err
 			}
-			out, err := outfactory.Parse(outputCfg)
+			storageBucket, out, err := outfactory.Parse(logger, outputCfg)
 			if err != nil {
 				return err
 			}
 
-			w, err := os.Create(*outFile)
+			// Store output file to a temp directory, then later upload it to object storage.
+			outFileName := path.Join(os.TempDir(), *outFile)
```

Comment:
Something is bit weird here, I think we can totally hide if output is file on filesystem, bytes buffer, object storage blob OR just HTTP/gRPC api response (: 

Let me propose a PR to this PR to show it, ok?

Reply:
that would be great, thanks!